src/ig/IGExporter.ts

```diff
@@ -46,16 +46,21 @@ export class IGExporter {
   private readonly outputLog: Map<string, outputLogDetails>;
   private readonly config: Configuration;
   private readonly configName: string;
+  private shouldCopyFiles: boolean;
   constructor(
     private readonly pkg: Package,
     private readonly fhirDefs: FHIRDefinitions,
     private readonly igDataPath: string,
-    private readonly isIgPubContext: boolean = false
+    private readonly isIgPubContext: boolean = false,
+    private readonly isLegacyIgPubContext: boolean = false
```

Comment:
If you assign a default variable, then I don't think you need to explicitly declare the type.  TS will infer it from the value.